src/main/java/no/unit/nva/bare/BareConnection.java

```diff
@@ -0,0 +1,30 @@
+package no.unit.nva.bare;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+
+public class BareConnection {
+
+    private static final transient String bareUrl = "https://authority.bibsys.no/authority/rest/functions/v2/query?";
```

Comment:
BARE_URL

Reply:
Yes, I think it should be BARE_URL but the checkstyle/pmd stuf complained about the capital ltter at the start and about a abbriviation for 3 or less letters in 'URL'.